src/parsers/images/bif.ts

```diff
@@ -77,7 +77,7 @@ function parseBif(buf : Uint8Array) : IBifObject {
 
   const aspectRatio = [buf[pos], buf[pos + 1]].join(":"); pos += 2;
 
-  const isVod = buf[pos] === 1; pos += 1;
+  const isVod = buf[pos] === 1;
```

Comment:
Hmm it doesn't change the logic here ?

Reply:
No. Because pos is set to another value just after.